### DISCUSSION TITLE
Text To Speech Repository

### DISCUSSION THREAD

**Discussion by wework13** [community/community#65784] [Repositories]
### Select Topic Area

Question

### Body

What is the best repository for converting voice to text in terms of power and speed?
I searched a lot, but I still don't know which one works better (it doesn't matter the language of the tool).

**Comment by amirroox** [comment]
In my opinion, the following repo:
https://github.com/coqui-ai/TTS

But it is better to check the link below and choose the tool according to your work.
https://github.com/topics/text-to-speech

**Also, avoid repetitive or unnecessary questions**

**Comment by Binary-Ninja-007** [comment]
👋 Hello! It seems you are looking for a repository for converting voice to text. While I'm primarily knowledgeable about text-to-speech (converting text to speech) rather than voice-to-text, I can provide some guidance based on popular options available on GitHub.

When it comes to voice-to-text conversion, some widely used repositories include:

Mozilla DeepSpeech: A popular open-source automatic speech recognition (ASR) engine developed by Mozilla. It provides pre-trained models for various languages and allows you to train your own models as well.

Kaldi: Kaldi is a toolkit for speech recognition that offers a wide range of features, including voice-to-text conversion. It provides efficient algorithms and supports various ASR techniques.

CMUSphinx: CMUSphinx is an open-source ASR system that offers both offline and online speech recognition capabilities. It supports several programming languages and has a large community.

Google Cloud Speech-to-Text: Although not an open-source repository, Google Cloud Speech-to-Text is a powerful cloud-based service that provides accurate voice recognition. It can handle various audio formats, transcribe audio in real-time, and has multilingual support.

Wit.ai: Wit.ai is a natural language understanding framework that includes speech recognition capabilities. It allows for voice-to-text conversion and offers pre-built models for different languages and domains.

It's important to note that the "best" repository for voice-to-text conversion might vary depending on your specific requirements, resources, and project constraints. I recommend exploring these repositories, their documentation, user reviews, and considering factors like performance, accuracy, ease of use, and community support to determine which one suits your needs best.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
